storage/internal/benchmarks/continuous_reads.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"math/rand"
+	"os"
+	"path"
+	"slices"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+	"golang.org/x/sync/errgroup"
+)
+
+// continuousReads is a benchmark that continuously downloads the same set of
+// objects from GCS until the timeout is reached.
+//
+// SET UP
+//   - Initialize structs to populate with the benchmark results.
+//   - Select a random API to use to upload and download the object, unless an
+//     API is set in the command-line,
+//   - Select a random size for the objects; this size will
+//     be between two values configured in the command-line.
+//   - Create a directory of objects of that size in GCS, uploading random
+//     contents directly from memory. The number of objects created is equal to
+//     the command-line configuration for the number of objects in a directory.
```

Comment:
I don't see a warmup step here. Can we add one?

Reply:
Do you want to have one that just does reads? Otherwise, I think the w1r3 warmups are sufficient.

Reply:
w1r3 warmups are fine, just wasn't clear to me that runs of this benchmark could be combined with the warmup option. Maybe just something to clarify in documentation.

Reply:
Done